2-mandatory/2-oxygen-levels.js

```diff
@@ -11,7 +11,15 @@
     Some string methods that might help you here are .replace() and .substring().
 */
 
-function findSafeOxygenLevel() {}
+function findSafeOxygenLevel(arr) {
+  let value = arr
+    .filter((item) => item[item.length - 1] === "%")
+    .map((item) => parseFloat(item))
+    .find((item) => item > 19.5 && item < 23.5);
```

Comment:
While this works, you're looping over the array 3 times. Can you do this problem with only using `find`?

Reply:
wow , perfect idea, thank you

Reply:
Done